protocol-designer/src/organisms/WellOrderModal/WellOrderVisualization.tsx

```diff
@@ -0,0 +1,42 @@
+import b2t_l2r from '../../assets/images/well-order_b2t-l2r.jpg'
+import b2t_r2l from '../../assets/images/well-order_b2t-r2l.jpg'
+import l2r_b2t from '../../assets/images/well-order_l2r-b2t.jpg'
+import l2r_t2b from '../../assets/images/well-order_l2r-t2b.jpg'
+import r2l_b2t from '../../assets/images/well-order_r2l-b2t.jpg'
+import r2l_t2b from '../../assets/images/well-order_r2l-t2b.jpg'
+import t2b_l2r from '../../assets/images/well-order_t2b-l2r.jpg'
+import t2b_r2l from '../../assets/images/well-order_t2b-r2l.jpg'
+
+import type { WellOrderOption } from '../../form-types'
+
+interface WellOrderVisualizationProps {
+  firstValue: WellOrderOption
+  secondValue: WellOrderOption
+}
+
+const imageMap: Record<string, string> = {
+  b2t_l2r,
+  b2t_r2l,
+  l2r_b2t,
+  l2r_t2b,
+  r2l_b2t,
+  r2l_t2b,
+  t2b_l2r,
+  t2b_r2l,
+}
+
+export function WellOrderVisualization(
+  props: WellOrderVisualizationProps
+): JSX.Element {
+  const { firstValue, secondValue } = props
+  const imageKey = `${firstValue}_${secondValue}`
+
+  return (
+    <img
+      height="100%"
+      width="274px"
```

Comment:
```suggestion
      height="274px"
      width="100%"
```

I think once you replace all images, the modal will be the same as the design.

The current images seem to have a lot of whitespace.
<img width="1935" alt="Screenshot 2024-10-28 at 10 56 18 AM" src="https://github.com/user-attachments/assets/ae2ef50e-2b71-4a5f-a64c-611d69e9c52a">